src/chapters/chapter1.md

```diff
@@ -1,19 +1,39 @@
 
-Amy must have heard the tires spinning on the loose gravel because when John looked up she was standing in the doorway. Under different circumstances she would have smiled. John failed to suppress his own smile which crept into the corners of his lips and inched them upwards as he neared the door. Hers was the first friendly face he'd seen in what seemed like weeks. He pulled the door open, dropped his duffel on the threshold and enveloped her slight frame in his massive arms, "How you doing, kid?"
+# Chapter 2
 
-Her body responded reflexively by sagging. Safe and secure in her big brother's arms, it rebelled by relaxing, something she'd not allowed herself since she'd first called the police to report their father was missing.
+It took a moment for his eyes to adjust to the light - or lack there of. No one had bothered to turn any lights on in the living room, so the only light was the little that managed to sneak past the tightly drawn curtains. His hand instinctively reached for the switch on the wall, but no lights came on.
 
-# Chapter 2
+"Burnt out."
+
+"Huh, strange. I know dad's always been mostly useless around the house, but to let all of the lights burn out and not change them feels... well, it feels unusual. The dust on the other hand - that's no surprise." Every horizontal surface was caked with dust. That's how it'd been ever since mom had passed - or would have been had it not been for Patrick, Amy, and him. Patrick had moved to New York years ago, and he hadn't stepped foot in the house for even longer, so taking care of dad had fallen on Amy - not that she didn't have plenty on her plate as it is.
+
+"Yeah, he's been even more _himself_ of late. Working all the time. It's been all I can do to get him to come over for Sunday dinner with me and Beau." Dad had always been a workaholic. Growing up he'd stay at the lab until well after the family had sat down for dinner. When he did finally get home, he'd head straight to his study, where he'd pour himself a whisky and light a cigar to bury himself in more work, but only after closing the big oak door.
+
+A light came on in the hallway, not all of them had burnt out apparently, and a few seconds and several footsteps later, Patrick stepped into view. Backlit from the hall light, his features were obscured, but, even after years of not seeing him or hearing his voice, there was no doubt who was standing in the doorway.
+
+"Amy said you'd come. I wasn't so sure."
+
+"Pat, please!"
+
+"No, no, it's alright." He said as reassuringly as he could to Amy before turning to face his little brother's outline, "I can't say I wasn't wondering the same about you."
+
+"Ha - I'm not the one who swore I'd never step foot in this house again!"
+
+"Guys! Now's not the time to get into a pissing match about something that happened over a decade ago! You can duke it out after we find dad. Until then, try to act like the grown men you are. For me. For dad."
```

Comment:
What flavor is Amy feeling about the boys' argument here? Angry? Annoyed? Impatient? Exasperated? Hurt? Tired?